apps/emqx/test/emqx_schema_tests.erl

```diff
@@ -930,3 +930,13 @@ timeout_types_test_() ->
             typerefl:from_string(emqx_schema:timeout_duration_s(), <<"4294967001ms">>)
         )
     ].
+
+unicode_template_test() ->
+    Sc = #{
+        roots => [root],
+        fields => #{root => [{template, #{type => emqx_schema:template()}}]}
+    },
+    HoconText = <<"root = {template = \"中文\"}"/utf8>>,
+    {ok, Hocon} = hocon:binary(HoconText),
+    _ = hocon_tconf:check_plain(Sc, Hocon),
```

Comment:
Nit: maybe check that `root` is indeed in the output; otherwise, if there was a typo, it wouldn't check the root key.